uni/lib/controller/local_storage/database/app_exams_database.dart

```diff
@@ -13,7 +13,7 @@ class AppExamsDatabase extends AppDatabase<List<Exam>> {
       : super('exams.db', [_createScript], onUpgrade: migrate, version: 6);
```

Comment:
Don't forget to update the databases versions in all modified files!

Reply:
Done!